recipes/devices/cuboxp.sh

```diff
@@ -83,21 +84,20 @@ EOF
 
 # Will be run in chroot - Post initramfs
 device_chroot_tweaks_post() {
-  log "Running device_chroot_tweaks_post" "ext"
-
-  log "Creating uInitrd from 'volumio.initrd'" "info"
-  #TODO This can be done outside chroot,
-  # removing the need of each image needing u-boot-tools
-  # saving some time!
-  mkimage -v -A arm64 -O linux -T ramdisk -C none -a 0 -e 0 -n uInitrd -d /boot/volumio.initrd /boot/uInitrd
-  log "Removing unnecessary /boot files"
-  rm /boot/volumio.initrd
-  log "Compiling u-boot boot script"
-  mkimage -C none -A arm -T script -d /boot/boot.cmd /boot/boot.scr
+  # log "Running device_chroot_tweaks_post" "ext"
+  :
 }
 
 # Will be called by the image builder post the chroot, before finalisation
 device_image_tweaks_post() {
-  # log "Running device_chroot_tweaks_post" "ext"
-  :
+  log "Running device_image_tweaks_post" "ext"
+  log "Creating uInitrd from 'volumio.initrd'" "info"
+  if [[ -f "${ROOTFSMNT}"/boot/volumio.initrd ]]; then
+    mkimage -v -A "${UINITRD_ARCH}" -O linux -T ramdisk -C none -a 0 -e 0 -n uInitrd -d "${ROOTFSMNT}"/boot/volumio.initrd "${ROOTFSMNT}"/boot/uInitrd
+    rm "${ROOTFSMNT}"/boot/volumio.initrd
+  fi
+  if [[ -f "${ROOTFSMNT}"/boot/boot.cmd ]]; then
+    log "Creating boot.scr"
+    mkimage -A "${UINITRD_ARCH}" -T script -C none -d "${ROOTFSMNT}"/boot/boot.cmd "${ROOTFSMNT}"/boot/boot.scr
+  fi
```

Comment:
The arch of the boot script ought to be 'arm', I have not seen 'arm64' anywhere (yet)

Reply:
Agree, oversight.. Will fix!

Reply:
Fixed in 3df87f0